src/main/java/it/pagopa/swclient/mil/util/Utility.java

```diff
@@ -0,0 +1,32 @@
+package it.pagopa.swclient.mil.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.pagopa.swclient.mil.controller.model.CommonHeader;
+
+import java.security.SecureRandom;
+import java.util.Base64;
+import java.util.stream.Collectors;
+
+public class Utility {
+    private Utility() {
+    }
+
+    public static String decodeBearerPayload(CommonHeader header) throws JsonProcessingException {
+        String[] chunks = header.authorization().split("\\.");
+        String payload = new String(Base64.getUrlDecoder().decode(chunks[1]));
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        JsonNode jsonNode = objectMapper.readTree(payload);
+
+        return jsonNode.get("sub").asText();
+    }
+
+    public static String generateTerminalUuid() {
```

Comment:
Secure random is good for generating UUID, but we usually prefer to pass through the UUID class, in particular by the UUID.randomUUID() that generates a UUID4 (This one also uses a secure random under the hood) What do you think about this? :)

Reply:
Resolved in [https://github.com/pagopa/mil-terminal-registry/pull/25/commits/8c55df2932810878f744b7992b7a535c9c819587](https://github.com/pagopa/mil-terminal-registry/pull/url)